smart_contracts/contracts_as/test/local-state/assembly/tsconfig.json

```diff
@@ -0,0 +1,6 @@
+{
+  "extends": "../../../../../../../.nvm/versions/node/v10.16.3/lib/node_modules/assemblyscript/std/assembly.json",
```

Comment:
This looks brittle...

Reply:
This file is copy-pasted from other AS contracts. If there's a better way to use tsconfig.json files I'd be happy to do a separate pass at addressing it in #1524

Reply:
When in rome I guess, thanks for answering. I suppose this is dependent on a certain dev environment setup for ts as well, one using nvm.